cms/core/tests/test_permissions.py

```diff
@@ -0,0 +1,110 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test import TestCase
+from wagtail.models import PAGE_PERMISSION_CODENAMES
+
+from cms.users.models import User
+
+WAGTAIL_PERMISSION_TYPES = ["add", "change", "delete"]
+
+WAGTAIL_PAGE_PERMISSION_TYPES = [codename.replace("_page", "") for codename in PAGE_PERMISSION_CODENAMES]
+
+
+class PermissionsTestCase(TestCase):
+    @classmethod
+    def create_user(cls, first_name, last_name, group_name):
+        # Create user
+        cls.user = User(first_name=first_name, last_name=last_name)
+        cls.user.save()
+
+        # Assign user to group
+        group = Group.objects.get(name=group_name)
+        group.user_set.add(cls.user)
+        group.save()
```

Comment:
You should not need `save` for adding m2m relations. That is only needed if you're creating a new instance (in this case a new Group instance)

https://docs.djangoproject.com/en/5.1/topics/db/examples/many_to_many/